File: FactoryPattern/Logistics/Products/Truck.cs
using Logistics.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Channels;
using System.Threading.Tasks;

namespace Logistics.Products
{
    public class Truck : ITransport
    {
        public double MaxKg => 20000;
        public string Deliver() => "Delivering by truck";
    }
}

File: FactoryPattern/Java/src/products/Truck.java
package products;
import interfaces.ITransport;

public class Truck implements ITransport {
    @Override
    public double getMaxCargoKg() {
        return 20000;
    }

    @Override
    public String deliver() {
        return "Доставка с КАМИОН по шосе";
    }
}
